### ISSUE TITLE
No indication of output directory if no output directory is given

### ISSUE THREAD

**Issue by bebraw** [Yomguithereal/kotatsu#116]
If you try to build without the output directory set (i.e. `kotatsu build client --babel --config ./webpack.config.js`), it doesn't indicate where it builds.

It could be a good idea to either disallow this or at least print where it built.

**Comment by Yomguithereal** [comment]
Yes, printing the output directory would be a good idea. Let me add this.
